example/src/TopCardsDisplayer.re

```diff
@@ -30,6 +40,31 @@ module SiteStatisticsFragment = [%relay.fragment
 let make = (~siteStatistics as siteStatisticsRef) => {
   let siteStatistics = SiteStatisticsFragment.use(siteStatisticsRef);
 
+  let environment = ReasonRelay.useEnvironmentFromContext();
+  React.useEffect0(() => {
+    let subscription =
+      CurrentVisitorsSubscription.subscribe(
+        ~environment,
+        ~variables=(),
+        ~onNext=
+          (response: CurrentVisitorsSubscription.Types.response) => {
```

Comment:
Just curious - did inference not work properly here?

Reply:
Oh, it worked perfectly, sorry. Just my habit from getting burned by type-inference-related problems in F# that kicks in, so now I annotate most functions. Removed this for consistency with the rest of the codebase.